db/migrate/20201116182332_create_bookings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBookings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookings do |t|
+      t.string :start_date
```

Comment:
According to [this stackoverflow post](https://stackoverflow.com/questions/4384284/ruby-on-rails-generates-model-fieldtype-what-are-the-options-for-fieldtype), we can actually encode this directly as a date.

For example, `rails g model Booking start_date:date`

This would probably be advantageous later on when it comes to working with the date itself, as we dont have to convert from/to a string all the time.

Reply:
I've converted this to date. Good shout!